checked_message.go

```diff
@@ -87,6 +112,8 @@ func (m *CheckedMessage) Write(fields ...Field) {
 	}
 
 	m.next.Write(fields...)
+	m.lvl, m.next, m.tail = invalidLevel, nil, nil
```

Comment:
Well, resetting at least the next and tail pointers are necessary before re-use; I'd rather get them "going" rather than "coming".

Especially now that the lack of atomic obviates the need for the entire pointer-value-stomp under NewCheckedMessage...